From: SeongJae Park <sj@kernel.org>
Date: Fri, 14 Feb 2025 17:22:56 -0800
Subject: [PATCH] ==== filter documentation fixup ====

Docs/mm/damon: misc DAMOS filters documentation fixes and improves

Fix and improve DAMOS filters documentation by fixing a copy-paste typo,
adding hugepage_size filter documentation on design doc, moving logic
details from usage to design, clarify DAMOS filters handling sequence
based on handling layer, and re-organizing the filters type list for
easier understanding of the handling sequence.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_haacx_e4 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_haacx_e4

diff --git a/hkml_cv_bogus/hkml_cv_bogus_haacx_e4 b/hkml_cv_bogus/hkml_cv_bogus_haacx_e4
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

